### ISSUE TITLE
Weird yellow rectangle on http://osmstreetlight.bplaced.net/

### ISSUE THREAD

**Issue by jidanni** [sb12/OSMStreetLight#28]
![Screenshot 2024-07-15 23 59 01](https://github.com/user-attachments/assets/7f6ee878-d73c-4dad-a154-146686e314e2)
Seen on http://osmstreetlight.bplaced.net/

**Comment by mkgrgis** [comment]
This is a figure for localization text like "Zoom in". Maybe in your version there is no such text.

**Comment by sb12** [comment]
Yes, it looks like an issue with the localized texts not loading. Could you please send me some more information which browser you used and what your browser language settings (e.g. screenshot of https://gtranslate.io/detect-browser-language) are, so I can try to reproduce it.

**Comment by sb12** [comment]
Thanks. I could reproduce the missing texts in Chrome incognito window. Not sure what causes it though...

**Comment by jidanni** [comment]
All I know is e.g., my https://www.jidanni.org/ does not have any information become inaccessible due to gambles that all components are working properly.

So you could perhaps first put the vital links on the page, and then only cover them up with better links if whatever system you are relying on can produce such links.

Else it seems like a fragile house of cards.

**Comment by sb12** [comment]
Thanks for your suggestion. I added some fallback text in case the localization completely fails.

Nevertheless there were multiple bugs in the implementation of the localization tool which are fixed now. 

Let me know in case there are still problems with the localization.
